fetcher/src/job/job.utils.ts

```diff
@@ -67,25 +71,33 @@ async function fetchRawDataWithoutProxy(adapter, logger) {
  * @param {} NestJs logger
  * @return {number} aggregatedresults
  */
-export async function fetchData(adapterList, logger) {
+export async function fetchData(adapterList, decimals, logger) {
```

Comment:
I personally think increasing function parameter numbers aren't good. adapter.decimals were previously only utilized from deviation checks not in fetchData. Is there a specific reason for dex fetcher to require this variable?

Reply:
For some pools, the tokens added are in different decimals. 
Eg. In wbtc USDT pool, WBTC Decimal is 8, oUsdt is 6. We need the decimal to convert to correct value

Reply:
I see, if it was integrated to reducers, this setting up decimals with corresponding pow would have been in the last step...

Reply:
Will be the next-to last step. We will need to convert is again to the Dec we save in our database

Reply:
sorry I couldn't understand, what do you mean by
`We will need to convert is again to the Dec we save in our database`

Reply:
if you look into the adaptor config, you can see we have 2 section to define decimal
1. decimal used by token0 and token1 in the pool
2. decimal defined by the adaptor,
So in some cases we need to do conversion twice
![image](https://github.com/Bisonai/orakl/assets/54656638/afd124e3-8c89-4cea-9d59-06583c46f856)

Reply:
Okie, thank you for the explanation